src/modules/ad/components/Ad.jsx

```diff
@@ -20,13 +20,30 @@ import useRobotsNoIndexMetaTag from "../../../common/hooks/useRobotsNoIndexMetaT
 import H1WithAutoFocus from "../../../common/components/h1WithAutoFocus/H1WithAutoFocus";
 import NotFound404 from "../../../common/components/NotFound/NotFound404";
 
-const Ad = ({ match }) => {
+/* eslint-disable no-underscore-dangle */
+function Ad({ match }) {
     const [{ data: ad, error, status }, dispatch] = useFetchReducer();
     const isInternal = match.path.startsWith("/stillinger/intern/");
     const avoidIndexing = (error && error.statusCode === 404) || (ad && ad._source.status !== "ACTIVE") || isInternal;
 
     useRobotsNoIndexMetaTag(avoidIndexing);
 
+    function fetchStilling(id) {
+        dispatch({ type: FetchAction.BEGIN });
+
+        const path = isInternal ? "intern" : "stilling";
+        SearchAPI.get(`api/${path}/${id}`).then(
+            (data) => {
+                dispatch({ type: FetchAction.RESOLVE, data });
+            },
+            // TODO: fix lint
+            // eslint-disable-next-line no-shadow
```

Comment:
Lint: no-shadow

Reply:
her er det to variabler i samme scope med samme navn `error`. Den ene lages på linje 25, den andre linje 41. Foreslår bare å rename den på linje 41 til `err`